eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,7 +1,5 @@
 variables:
   DocWardenVersion: '0.6.1'
-  DotNetCoreSDKVersion: '3.1.301'
```

Comment:
Why remove these? The agent configuration will change. These make sure we get the same baseline as we want to support for our customers.

Reply:
Its now specified in the [global.json](https://github.com/Azure/azure-sdk-for-net/blob/6171954d8a9ade7ed5b20b71afa52ce609b6fa71/global.json#L6)

Reply:
That just required it. Something has to install it, which is what those tasks do.

Reply:
They are already installed on the build agents. so Installing it again is quite redundant.

Reply:
According to their [documentation](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/UseDotNetV2#user-content-overview:~:text=If%20the%20.NET%20Core%20version%20you,right%20version%20installed%20on%20the%20agent.), it shouldn't acquire a version if already installed; however, I can find nothing in their source that supports that claim.

Reply:
Part of the reason we have made this change is to avoid it downloading and installing the sdk every time as that step fails from time to time which we want to try and eliminate. See https://github.com/Azure/azure-sdk-for-net/issues/12120 for some of that conversation.